src/mapReducers.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import { isTargetAllAction } from './context';
```

Comment:
Do we consider that action as "natural" for map reducers? Action to target all mapped reducers.

I think yes, and for a natural behavior nothing should be implemented additionally.

It makes more sense to add option to disable `AllAction` then to enable it every time. It is a lot easier to know that you do not want your reducer to get affected by `AllAction` then to know if it should handle `AllAction`.

You can not predict who will trigger `AllAction` and with what parameters.